src/actions/appInstance.js

```diff
@@ -56,6 +56,11 @@ const getAppInstance = async () => async (dispatch, getState) => {
 
     const appInstance = await response.json();
 
+    // set default selected space if is not defined
+    if (!appInstance.settings.spaces && spaceId) {
```

Comment:
Maybe also add the case in which `[]` is empty? Or add this directly to the reducer as a default.

Reply:
I won't add this case to allow *not* selecting the current space.